sylvia-derive/src/message.rs

```diff
@@ -730,12 +776,12 @@ impl<'a> MsgVariants<'a> {
                     }
                 }
 
-                impl <'a, C: #sylvia ::cw_std::CustomQuery> Querier for BoundQuerier<'a, C> {
+                impl <'a, C: #sylvia ::cw_std::CustomQuery, #(#unbonded_generics,)*> #querier for BoundQuerier<'a, C> #where_clause {
                     #(#methods_impl)*
                 }
 
 
-                pub trait Querier {
+                pub trait #querier #where_clause {
```

Comment:
Really needed? 90% of cases you don't want to have where clauses on this level. The best is to keep them only on impl level, or on function definition level. Anywhere else is only when there is very good explanation for that.

Reply:
You are right. This is not needed